lib/ex_aws/request/httpoison.ex

```diff
@@ -0,0 +1,14 @@
+defmodule ExAws.Request.HTTPoison do
+  @behaviour ExAws.Request.HttpClient
+
+  @moduledoc false
+
+  @overriding_opts [
+    recv_timeout: 5_000,
+    hackney: [pool: false]
+  ]
```

Comment:
@mschae Do you know a good way to verify that a request is made without using the pool?